delocate/delocating.py

```diff
@@ -148,7 +190,12 @@ def copy_recurse(lib_path, copy_filt_func=None, copied_libs=None):
     return copied_libs
 
 
-def _copy_required(lib_path, copy_filt_func, copied_libs):
+def _copy_required(
```

Comment:
Note that this function will be orphaned by removal of deprecated `copy_recurse`, and removed with it?